tests/openedx_tagging/core/tagging/test_system_defined.py

```diff
@@ -0,0 +1,236 @@
+""" Test the System-defined taxonomies and object tags """
+
+import ddt
+
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.test.testcases import TestCase, override_settings
+
+from openedx_tagging.core.tagging.system_defined_taxonomies.object_tags import (
+    ModelObjectTag,
+    UserObjectTag,
+    LanguageObjectTag,
+)
+from openedx_tagging.core.tagging.models import Tag, Taxonomy
+
+from .test_models import TestTagTaxonomyMixin
+
+test_languages = [
+    ('en', 'English'),
+    ('az', 'Azerbaijani'),
+    ('id', 'Indonesian'),
+    ('qu', 'Quechua'),
+    ('zu', 'Zulu'),
+]
+
+
+class EmptyTestClass:
+    """
+    Empty class used for testing
+    """
+
+
+class EmptyTestModel(models.Model):
+    """
+    Model used for testing
+    """
+    mi_id = models.AutoField(primary_key=True)
+
+    class Meta:
+        managed = False
+        app_label = 'oel_tagging'
+
+
+class EmptyModelObjectTag(ModelObjectTag):
+    """
+    Model ObjectTag used for testing
+    """
+
+    system_defined_taxonomy_id = 3
+
+    class Meta:
+        proxy = True
+        managed = False
```

Comment:
Cool.. I didn't know you could do this in tests!

Reply:
Thanks :smile: